index.js

```diff
@@ -3,32 +3,257 @@ const dropDown = document.getElementById('drop-down');
+      seeSource.href = item.source;
+      seeSource.innerHTML = 'See Source <i class="fa-brands fa-github"></i>';
+      modalBtns.appendChild(seeSource);
+
+      container.appendChild(modalBtns);
+      split.appendChild(container);
+      popContent.appendChild(split);
+      popUp.appendChild(popContent);
+      hi = document.body.appendChild(popUp);
+    }
+    return hi;
+  });
 };
 
-portfolioLink.addEventListener('click', removeModal);
-aboutLink.addEventListener('click', removeModal);
-contactLink.addEventListener('click', removeModal);
\ No newline at end of file
+projectLists.map((item) => {
+  const projectItem = document.createElement('div');
+  projectItem.className = 'project-item';
+
+  const projectImage = document.createElement('div');
+  projectImage.className = 'project-image';
+  const image = document.createElement('img');
+  image.src = item.imageLink;
+  image.alt = item.alt;
+  projectImage.appendChild(image);
+
+  const projectDetails = document.createElement('div');
+  projectDetails.className = 'project-details';
+  const projName = document.createElement('h3');
+  projName.innerText = item.name;
+  projectDetails.appendChild(projName);
+
+  const canopy = document.createElement('p');
+  canopy.className = 'canopy-para';
+  const firstSpan = document.createElement('span');
+  firstSpan.innerText = 'CANOPY';
+  canopy.appendChild(firstSpan);
+  const fstdotImage = document.createElement('img');
+  fstdotImage.src = './assets/Counter.png';
+  fstdotImage.alt = 'a dot icon';
+  canopy.appendChild(fstdotImage);
+  const secondSpan = document.createElement('span');
+  secondSpan.innerText = 'Back end dev';
+  canopy.appendChild(secondSpan);
+  const snddotImage = document.createElement('img');
+  snddotImage.src = './assets/Counter.png';
+  snddotImage.alt = 'a dot icon';
+  canopy.appendChild(snddotImage);
+  const thirdSpan = document.createElement('span');
+  thirdSpan.innerText = '2015';
+  canopy.appendChild(thirdSpan);
+  projectDetails.appendChild(canopy);
+
+  const projectDes = document.createElement('p');
+  projectDes.className = 'mini-description';
+  projectDes.innerText = item.description;
+  projectDetails.appendChild(projectDes);
+
+  const languageTags = document.createElement('ul');
+  languageTags.className = 'language-tags';
+  const fListItem = document.createElement('li');
+  const [one, two, three] = item.tech;
+  fListItem.innerText = one;
+  languageTags.appendChild(fListItem);
+  const sListItem = document.createElement('li');
+  sListItem.innerText = two;
+  languageTags.appendChild(sListItem);
+  const tListItem = document.createElement('li');
+  tListItem.innerText = three;
+  languageTags.appendChild(tListItem);
+  projectDetails.appendChild(languageTags);
+
+  const projectBtn = document.createElement('a');
+  projectBtn.style.textDecoration = 'none';
+  projectBtn.href = '#pop-head';
+  projectBtn.className = 'project-detail-btn';
+  projectBtn.innerHTML = 'See Project';
```

Comment:
- [ ] Based on the requirements of this project, when the user clicks (or taps) on this button to check project details, the popup with details about the project appears. Right now, there's no popup. Let's kindly fix this issue.

Reply:
Hello @FrederickMih thanks for the review but the pop-up works. kindly cross check.
please use the git-hub page below to preview it live. 
[my portfolio](https://trust-007.github.io/My-Portfolio-Website/#pop-head)